src/controllers/trainee/Controller.ts

```diff
@@ -11,79 +14,169 @@ class TraineeController {
       return TraineeController.instance;
     }
 
-    get(req: Request, res: Response, next: NextFunction) {
-      try {
-        console.log('Inside get method of Trainee Controller');
-      res.send({
-          message: 'Trainees fetched successfully',
-          data: [
-            {
-              name: 'Trainee1',
-              address: 'Noida'
-            }
-          ]
-        });
-      } catch (err) {
-        console.log('Inside err', err);
-        next({
-          error: 'Error Occured in fetching user',
-          code: 500,
-          message: err
-        });
+    public async getAll(req: IRequest, res: Response, next: NextFunction) {
+
+      let skip: number;
+      let limit: number;
+      let sort: boolean;
+      let search: string = '';
+
+      if ('limit' in req.query) {
+          limit = Number(req.query.limit);
+      }
+      else {
+          limit = 10;
```

Comment:
This could be a one-liner using a conditional operator like 

`let limit = ('limit' in req.query) ? Number(req.query.limit) : 10;`